Library/Homebrew/metafiles.rb

```diff
@@ -1,13 +1,15 @@
+require "set"
+
 class Metafiles
   # https://github.com/github/markup#markups
   EXTENSIONS = %w[
     .adoc .asc .asciidoc .creole .html .markdown .md .mdown .mediawiki .mkdn
     .org .pod .rdoc .rst .rtf .textile .txt .wiki
-  ].freeze
+  ].to_set.freeze
```

Comment:
Keep in mind that `Keg#text_files` queries this collection once for every file, so bringing the complexity of that lookup down is worthwhile if we care about `brew bottle` runtime.